### PR TITLE
add send_image_url, add appsecret_proof option, fix for issue #6

### PR DESCRIPTION
**Pull request by tbsf**
This is similar to send_image but takes a URL image location instead of a filename.

### CODE REVIEW

File: pymessenger/bot.py [hunk 1]
```diff
@@ -5,14 +5,18 @@
...
-    def __init__(self, access_token, api_version=DEFAULT_API_VERSION):
+    def __init__(self, access_token, api_version=DEFAULT_API_VERSION, app_secret=None):
         self.api_version = api_version
         self.access_token = access_token
         self.base_url = (
             "https://graph.facebook.com"
             "/v{0}/me/messages?access_token={1}"
         ).format(self.api_version, access_token)
 
+        if app_secret is not None:
```
Comment (tbsf) [inline pymessenger/bot.py]: Please read more about appsecret_proof here:

https://developers.facebook.com/docs/graph-api/securing-requests#appsecret_proof

File: pymessenger/bot.py [hunk 2]
```diff
@@ -102,3 +107,31 @@ def send_image(self, recipient_id, image_path):
...
+        payload = {
+            'recipient': json.dumps(
+                {
+                    'id': recipient_id
+                }
+            ),
+            'message': json.dumps(
+                {
+                    'attachment': {
+                        'type': 'image',
```
Comment (tbsf) [inline pymessenger/bot.py]: Read more about sending an image_url here:

https://developers.facebook.com/docs/messenger-platform/send-api-reference

File: pymessenger/bot.py (hunk 1 above)
Comment (davidchua) [inline pymessenger/bot.py]: I like this! This is definitely something we should have added into the wrapper at the start! :+1:

**Comment by davidchua** [comment]
Thanks @tbsf for the PR!

Sorry I haven't been able to keep an eye on this as often as I should. Your contribution is amazing!

Would you be interested in co-administrating this project?

**Comment by tbsf** [comment]
@davidchua Absolutely, I'd be glad to help out :)

**Comment by davidchua** [comment]
@tbsf Added you as a collaborator :)

**Comment by tbsf** [comment]
@saikiran2602 https://developers.facebook.com/docs/messenger-platform/send-api-reference

**Pull request merged**